src/components/tooltip/Tooltip.tsx

```diff
@@ -380,6 +390,25 @@ class Tooltip extends React.Component<TooltipProps, State> {
             </div>
         );
 
+        if (shouldTrapFocus) {
+            tooltip = (
```

Comment:
it looks like `shouldTrapFocus` cannot be used with `stopBubble`, is that okay? do we need to wrap the existing value?

```
if (shouldTrapFocus) {
    tooltip = (
        <FocusTrap aria-modal shouldDefaultFocus>{tooltip}</FocusTrap>
    );
}
```

Reply:
yes, that's a good point, I will modify